docs/developers/contributing/documentation/index.md

```diff
@@ -498,7 +522,9 @@ make slimgallery-live
 ```
 These builds will build the documentation with the entire gallery. The [`-live`
 variant](live-builds) will will open a browser preview and auto-rebuild any pages you edit.
+````
 
+````{dropdown} Single example in the gallery
```

Comment:
Based on the logic above, but I dont love it.
```suggestion
````{dropdown} napari/docs and a single example in the gallery
```